src/hb-open-file-private.hh

```diff
@@ -299,6 +501,7 @@ struct OpenTypeFontFile
     CFFTag		= HB_TAG ('O','T','T','O'), /* OpenType with Postscript outlines */
     TrueTypeTag	= HB_TAG ( 0 , 1 , 0 , 0 ), /* OpenType with TrueType outlines */
     TTCTag		= HB_TAG ('t','t','c','f'), /* TrueType Collection */
+    DFontTag		= HB_TAG ( 0 , 0 , 1 , 0 ), /* DFont Mac Resource Fork */
```

Comment:
Obviously this is not standard but every .dfont file I could see have it. If we don't agree on it we should provide a different API for whole dfont thing.